tests/s3_s3express_client_test.c

```diff
@@ -256,14 +256,102 @@ static int s_s3express_client_put_test_real_server_helper(struct aws_allocator *
     return AWS_OP_SUCCESS;
 }
 
-TEST_CASE(s3express_client_put_test_small_real_server) {
+TEST_CASE(s3express_client_put_test_small) {
     (void)ctx;
-    return s_s3express_client_put_test_real_server_helper(allocator, MB_TO_BYTES(1));
+    return s_s3express_client_put_test_helper(allocator, MB_TO_BYTES(1));
 }
 
-TEST_CASE(s3express_client_put_test_large_real_server) {
+TEST_CASE(s3express_client_put_test_large) {
```

Comment:
Debatable: I wouldn't call 100MB large. Perhaps s3express_client_put_mpu_test would be better. Sometimes I specifically search for tests related to MPU / non-MPU, etc., and this naming would make it easier to find them.